tokens/custom/medium-vars.css

```
@@ -129,4 +129,9 @@
 	/* XXL and XXS icon sizes are not within the design spec and are planned to be deprecated in Spectrum 2. */
 	--spectrum-workflow-icon-size-xxl: 32px;
 	--spectrum-workflow-icon-size-xxs: 12px;
+
+	--spectrum-in-field-button-side-edge-to-fill-small: 3px;
+	--spectrum-in-field-button-side-edge-to-fill-medium: 4px;
+	--spectrum-in-field-button-side-edge-to-fill-large: 4px;
+	--spectrum-in-field-button-side-edge-to-fill-extra-large: 4px;
```

```suggestion
--spectrum-in-field-button-side-edge-to-fill-small: 4px;
	--spectrum-in-field-button-side-edge-to-fill-medium: 3px;
	--spectrum-in-field-button-side-edge-to-fill-large: 4px;
	--spectrum-in-field-button-side-edge-to-fill-extra-large: 4px;
```

These tokens are listed differently in Figma. I think they must have changed where both sets are the same. But these are supposed to be different from the values in large-vars.css, correct?